ui/src/components/markdown-textarea.tsx

```diff
@@ -121,7 +122,7 @@ export class MarkdownTextArea extends Component<
               required
               disabled={this.props.disabled}
               rows={2}
-              maxLength={10000}
+              maxLength={this.props.maxLength || 1000}
```

Comment:
Do || `10000`, many comments and sidebars are going to be a lot longer than 1000 chars.